examples/k8s/github-job.yaml

```diff
@@ -10,21 +10,21 @@ spec:
       - name: provisioner
         image: alpine/git
         args:
-        - clone 
+        - clone
         - https://github.com/jpillora/chisel
         - /makisu-context
         volumeMounts:
         - name: context
           mountPath: /makisu-context
       containers:
       - name: makisu
-        image: gcr.io/makisu-project/makisu:v0.1.11
+        image: gcr.io/uber-container-tools/makisu:v0.1.11
         imagePullPolicy: IfNotPresent
         args:
         - build
         - --push=gcr.io
         - --modifyfs=true
-        - -t=makisu-project/example-github:v0.1.11
```

Comment:
We should update the version numbers in the examples to match images that do exist in the new repository. Alternatively, we should do a backfill.

Reply:
Will do both.